app/database_etl/etl_main.py

```diff
@@ -37,19 +36,22 @@ def main():
         # Get all records with airtable API request and load into dataframe
         json = get_all_records()
         etl_report.set_num_airtable_records(len(json))
-        data = pd.DataFrame(json)
+        airtable_master_data = pd.DataFrame(json)
 
         # Clean raw airtable records to standardize data formats
-        data = standardize_airtable_data(data)
+        airtable_master_data = standardize_airtable_data(airtable_master_data)
+
+        # Add test adjustment data
+        airtable_master_data = add_test_adjustments(airtable_master_data)
```

Comment:
Let's add some reporting here. For now, let's at least report how many estimates need test adjustment (may be useful to further split into the number of new records and the number of modified records)